src/components/SurveyList/SurveyList.module.css

```
@@ -0,0 +1,8 @@
+.swiperPagination {
+  /* Override default Swiper style */
+  bottom: 40px !important;
+}
```

```suggestion
.swiper-pagination-bullets.swiper-pagination-horizontal {
  /* Override default Swiper style */
  bottom: 40px;
}
```

or if we want to be more generic:

```suggestion
.swiper-pagination.SurveyList_swiperPagination__1P3sR.swiper-pagination-bullets.swiper-pagination-horizontal {
  /* Override default Swiper style */
  bottom: 40px;
}
```

This suggestion is not perfect:
- on the pros, we avoid using `!important`, which whenever possible is still good to have :) 
- on the cons, we add a bit more coupling in the name of css classes (there is coupling already with the `swiper-pagination` class, but we just add more. This increases the chances of breaking changes.

IMO I prefer avoiding the `!important` but I would understand that other people disagree :D

---

I removed the `!important` here as the CSS module is using here so we can't specify the class name in the CSS module. So I override the pagination style from the library by adding a parent selector here [937c6aa](https://github.com/hanam1ni/react-surveyor/pull/72/commits/937c6aa3c5078def206bb1e00e913ca41ed027bb)